packages/calcite-components/src/custom-theme.stories.ts

```diff
@@ -77,33 +89,52 @@ export const themingInteractive = (args: Record<string, string>): string => {
       }
     </style>
     <div class="demo">
-      <div class="demo-column">
-        ${accordion}
-        ${notices}
-        ${segmentedControl}
-        ${icon}
-      </div>
-      <div class="demo-column">
-        <div>
-          ${card}
+        <div class="demo-column">
+          ${accordion} ${actionBar} ${notices} ${segmentedControl}
+          <div style="display: flex">
+            ${actionPad}
+            <div style="width: 40px; height: 40px;">${actionMenu}</div>
+            ${icon}
+          </div>
         </div>
-        <div>
-          ${dropdown}
-          ${buttons}
+        <div class="demo-column">
+          <div>${card}</div>
+          <div>${dropdown} ${buttons}</div>
+          <div>${checkbox}</div>
+          ${chips} ${pagination} ${slider}
         </div>
-        <div>
-          ${checkbox}
-        </div>
-        ${chips}
-        ${pagination}
-        ${slider}
-      </div>
-      <div class="demo-column">
-        ${datePicker}
-        ${tabs}
-        ${loader}
-        ${calciteSwitch}
+        <div class="demo-column">${datePicker} ${tabs} ${loader} ${calciteSwitch}</div>
       </div>
     </div>
   </div>`;
+
+export default {
+  title: "Theming/Custom Theme",
+  args: {
+    ...globalTokens,
+    ...actionTokens,
+    ...actionBarTokens,
+    ...actionMenuTokens,
+    ...actionPadTokens,
+    ...actionGroupTokens,
+    ...cardTokens,
+  },
+};
+
+export const themingInteractive = (args: Record<string, string>): string => {
+  return kitchenSink(args);
+};
+
+export const theming_TestOnly = (): string => {
```

Comment:
Late to the party, but new screenshot tests don't need the `_TestOnly` suffix anymore.

Reply:
I'd like to somehow distinguish between the story that is designed to allow the user to change the variables vs variables that are set to a new color theme for testing purposes already.

Reply:
That's fine, but we're no longer using that specific suffix (see https://github.com/Esri/calcite-design-system/issues/9085 for context). Calling this out shouldn't be necessary since most stories are primarily used for screenshot tests.

I think the name for the interactive story already conveys the separation well, and we could consider adopting this approach for other interactive stories too.